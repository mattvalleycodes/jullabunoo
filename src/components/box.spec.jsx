import * as React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import Box from "./box";

describe("<Box /> component", () => {
  test("renders the provided component", () => {
    const box = renderer.create(<Box>FOO!</Box>).toJSON();

    expect(box).toMatchSnapshot();
  });

  test("attaches the provided CSS classes to the DOM", () => {
    const message = shallow(<Box className="foo">Bar!</Box>);

    expect(message.hasClass("foo")).toEqual(true);
  });
});
