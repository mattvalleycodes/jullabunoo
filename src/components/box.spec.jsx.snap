// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> component renders the provided component 1`] = `
<div
  className="box"
>
  FOO!
</div>
`;
